packages/design-tokens/src/scss/_colors.scss

```diff
@@ -58,7 +54,7 @@ $brand-colors: (
 
 // Background
 $background-basic: $white !default;
-$background-cover: $porcelain-50 !default;
+$background-cover: $tower-gray-50 !default;
 $background-inverted: $brand-primary !default;
```

Comment:
Apart from interaction states below (where the benefit of sharing `action` with `action-link` is clear), I think `background-inverted` should be linked directly to `gable-green-800`. But maybe the designers have told you otherwise…

Reply:
I checked it with designers and we decided to split it. So background inverted is now set to gable-green color directly.